packages/run-protocol/src/vaultFactory/liquidation.js

```diff
@@ -42,15 +41,14 @@ const liquidate = async (zcf, vault, burnLosses, strategy, collateralBrand) => {
     strategy.keywordMapping(),
     strategy.makeProposal(collateralToSell, runDebt),
     vaultSeat,
-    liquidationSeat,
   );
   trace(` offeredTo`, collateralToSell, runDebt);
 
   // await deposited, but we don't need the value.
   await Promise.all([deposited, E(liqSeat).getOfferResult()]);
 
   // Now we need to know how much was sold so we can pay off the debt
-  const runProceedsAmount = liquidationSeat.getAmountAllocated('RUN', runBrand);
+  const runProceedsAmount = vaultSeat.getAmountAllocated('RUN', runBrand);
```

Comment:
I would add a comment here that we can use this because only liquidation adds RUN to the vaultSeat.